cucumber/dev-base/linux-headers/linux-headers.buildinfo

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2018 Scott Court, Maarten Hendrickx
```

Comment:
Usually I do this like:
``
# Copyright 2016 Scott Court
# Copyright 2018 Maarten Hendrickx
``

Reply:
I will try to do this from now